master/docs/tutorial/tour.rst

```diff
@@ -203,85 +205,61 @@ Let's tell the bot to notify certain events, to learn which EVENTS we can notify
-
-  c['status'].append(irc.IRC(host="irc.freenode.org", nick="bbtest",
-                             allowForce=True,
-                             channels=["#buildbot-test"]))
-
-This time, the bot is giving you more output, as it's specifically responding to your direct request to force a build, and explicitly tells you when the build finishes:
+Now, go back to the web interface and force another build. Alternatively, ask
+the bot to force a build:
 
 .. code-block:: irc
 
-  <@lsblakk> bbtest: force build runtests test build
-  < bbtest> build #2 of runtests started, including []
-  < bbtest> build forced [ETA 0 seconds]
-  < bbtest> I'll give a shout when the build finishes
-  < bbtest> build #2 of runtests is complete: Success [build successful]  Build details are at http://localhost:8010/builders/runtests/builds/2
+  <@lsblakk> bbtest: force build --codebase= runtests
+  < bbtest> build #1 of runtests started
+  < bbtest> Hey! build runtests #1 is complete: Success [finished]
 
 You can also see the new builds in the web interface.
 
 .. image:: _images/irc-testrun.png
    :alt: a successful test run from IRC happened.
 
+The full documentation is available at :ref:`IRC-Bot`.
+
 Setting Authorized Web Users
 ----------------------------
 
-Further down, look for the WebStatus configuration::
-
-   c['status'] = []
-
-   from buildbot.status import html
-   from buildbot.status.web import authz, auth
-
-   authz_cfg=authz.Authz(
-       # change any of these to True to enable; see the manual for more
-       # options
-       auth=auth.BasicAuth([("pyflakes","pyflakes")]),
-       gracefulShutdown = False,
-       forceBuild = 'auth',  # use this to test your worker once it is set up
-       forceAllBuilds = False,
-       pingBuilder = False,
-       stopBuild = False,
-       stopAllBuilds = False,
-       cancelPendingBuild = False,
-   )
-   c['status'].append(html.WebStatus(http_port=8010, authz=authz_cfg))
-
-The ``auth.BasicAuth()`` define authorized users and their passwords.
-You can change these or add new ones.
+The default configuration allows everyone to perform any task like creating or
+stopping builds via the web interface. To restrict this to a user, look for::
+
+  c['www'] = dict(port=8010,
+                   plugins=dict(waterfall_view={}, console_view={}))
+
+and append::
+
+  class RolesFromUsername(RolesFromBase):
+      def __init__(self, role, usernames):
+          self.role = role
+          self.usernames = usernames
+
+      def getRolesFromUser(self, userDetails):
+          if 'username' in userDetails:
+              username = userDetails['username']
+              if username in self.usernames:
+                  return self.roles
+          return []
+
+  c['www']['authz'] = util.Authz(
+          allowRules = [
+              util.AnyEndpointMatcher(role="users")
+          ],
+          rolematcher = [
+                RolesFromUsername('users', ['Alice'])
```

Comment:
probably the example should say Admin instead of Alice, because that user is really the admin